sdk/python/kfp/containers/_build_image_api.py

```diff
@@ -111,11 +111,22 @@ def build_image_from_working_dir(image_name: str = None, working_dir: str = None
             dockerfile_text = _generate_dockerfile_text(context_dir, dst_dockerfile_path, base_image)
             with open(dst_dockerfile_path, 'w') as f:
                 f.write(dockerfile_text)
+        
+        from ._cache import calculate_recursive_dir_hash, try_read_value_from_cache, write_value_to_cache
```

Comment:
nit: Could you move this import to the top of the file?

Reply:
I'll move it.

Reply:
Done.